.github/workflows/ci.yml

```diff
@@ -36,8 +36,15 @@ jobs:
       - name: Build with Cabal
         run: cabal build
 
-      # Download and untar the required file
-      - name: Download and untar PLONK params
+      # Cache PLONK params download
+      - name: Cache PLONK params
+        uses: actions/cache@v3
+        with:
+          path: ~/.sp1/circuits/plonk_bn254/v1.0.8-testnet
+          key: plonk-params-v1.0.8-testnet
```

Comment:
Is there a way we can make the version variable going forward, in order to cache the params beyond `v1.0.8-testnet`? One way could be fetching the version at https://github.com/argumentcomputer/lurk-hs/blob/main/src/lib.rs#L83